### ISSUE TITLE
mechine learning

### ISSUE THREAD

**Issue by elimsc** [elimsc/notes#23]
supervised
不是基于逻辑得到答案，而是基于数据与*统计*得到答案

## framework/toolkit
tensorflow
panda: DataFrame,Series
![image](https://user-images.githubusercontent.com/13717100/58254355-22481e80-7d9d-11e9-9ea4-78d7cbb8e966.png)

numpy

**Comment by elimsc** [comment]
**model**. In supervised learning, a machine learning algorithm builds a model by examining many examples and attempting to find a model that minimizes loss; this process is called *empirical risk minimization*.
**loss function**: L2 loss `(y - y1)^2`
